.github/workflows/ci-uto.yaml

```diff
@@ -4,6 +4,7 @@ on:
     branches: [ 'main', 'release-**' ]
     paths:
       - 'universal_transfer_operator/**'
+      - '.github/workflows/ci-uto.yaml'
```

Comment:
This corresponding path for tracking the changes in `ci-sql-cli.yaml` file and triggering the CI is missing in `.github/workflows/ci-sql-cli.yaml`. Can we please also add that in this PR ?

Reply:
Done.